src/gui/menu_configuration.cpp

```diff
@@ -136,3 +137,24 @@ void DissolveWindow::on_ConfigurationExportToXYZAction_triggered(bool checked)
     else
         Messenger::print("Successfully exported configuration '{}' to '{}'.\n", cfg->name(), fileAndFormat.filename());
 }
+
+void DissolveWindow::on_ConfigurationAdjustTemperatureAction_triggered(bool checked)
+{
+    // Get the currently-displayed Configuration
+    auto *cfg = ui_.MainTabs->currentConfiguration();
+    if (!cfg)
+        return;
+
+    auto ok = false;
+    auto temperature = QInputDialog::getDouble(this, "Set temperature for configuration",
+                                               "Enter the temperature (K) to apply to this configuration", cfg->temperature(),
+                                               -10000.0, 10000.0, 3, &ok);
```

Comment:
Well, that's breaking the laws of physics!  :smiley:
```suggestion
                                               0.0, 1000000.0, 3, &ok);
```

Reply:
Just keeping you on your toes!